laterpay/application/Controller/Admin/Post/Metabox.php

```diff
@@ -33,11 +33,11 @@ public static function get_subscribed_events() {
             ),
             'laterpay_admin_enqueue_styles_post_edit' => array(
                 array( 'laterpay_on_admin_view', 200 ),
-                array( 'load_assets' ),
+                array( 'load_assets', 100 ),
```

Comment:
Did `load_assets` run before `laterpay_on_admin_view`, is that why this change is added?

Reply:
I have change the priority. because it need to run before `LaterPay_Controller_Admin_TinyMCE::localize_script()`
https://github.com/rtCamp/laterpay-wordpress-plugin/pull/23/files#diff-242d3af52dab289cf11c4e6fcfee7c5fR83

So, script will already enqueued before we localize data.